swim/ping_handler.go

```diff
@@ -40,8 +42,11 @@ func handlePing(node *Node, req *ping) (*ping, error) {
 	changes, fullSync :=
 		node.disseminator.IssueAsReceiver(req.Source, req.SourceIncarnation, req.Checksum)
 
+	// Bidirectional full sync.
 	if fullSync {
-		// TODO: handle full sync
+		go func() {
```

Comment:
I don't think it will full-sync all the time: if checksums match, it will return false: https://github.com/uber/ringpop-go/blob/6e836b4043b3171b651a803024677cca38135898/swim/disseminator.go#L165

However, I agree with the second argument: we should throttle the number of goroutines we spawn here, possibly one.